src/apps/material/material.test.ts

```diff
@@ -35,9 +35,53 @@ describe("Material", () => {
     cy.contains("reserver");
   });
 
+  it("Open modal by clicking on reserver button (reserver Dummy bog) and close it with the x bottom", () => {
+    cy.contains("button:visible", "reserver Dummy bog").click();
+    cy.contains("Afhentes på");
+    cy.contains("Hovedbiblioteket");
+    cy.contains("22403525");
+    cy.contains("kasperbirch1@gmail.com");
+    cy.get(`[aria-label="Luk reservation modal"]`).click({
+      multiple: true,
+      force: true
+    });
```

Comment:
Consider how you can drop the `multiple/force` element. You probably want to use the visible selector here again.